Orange/widgets/data/owtranspose.py

```diff
@@ -33,6 +57,7 @@ class Outputs:
     feature_type = ContextSetting(GENERIC)
     feature_name = ContextSetting("")
     feature_names_column = ContextSetting(None)
+    remove_redundant_inst = ContextSetting(False)
```

Comment:
Wouldn't a True be a better default? If we migrate so that it becomes False for already saved workflows we are not breaking anything.